site/content/en/docs/manual/basics/cloud-storages.md

```diff
@@ -0,0 +1,18 @@
+---
+title: 'Cloud storages page'
+linkTitle: 'Cloud storages page'
+weight: 20
+description: 'Overview of the cloud storages page.'
+---
+
+![](/images/image227.jpg)
```

Comment:
I guess It's better to use [svg](https://github.com/ant-design/ant-design-icons/blob/master/packages/icons-svg/svg/outlined/cloud-sync.svg) than the screenshot. What do you think? Also, I don't see it on the website with documentation. This [link](https://tosmanov.github.io/cvat/docs/manual/basics/cloud-storages/) was used. Am I doing something wrong?

Reply:
I fixed the preview of the site, there was a bug. I think using svg in this case is not convenient because for the correct display need to specify the styles